wiki/fr/fr-Software-Manual.md

```diff
@@ -0,0 +1,226 @@
+
+![Fenêtre du Délai de temporisation](https://user-images.githubusercontent.com/20726856/97361808-65ff5b80-18a0-11eb-88d6-fb2131f10c75.png)
+
+Sous Linux, utilisez l'outil de configuration Jack pour modifier la taille de la mémoire tampon.
+
+Le délai actuel de la mémoire tampon a une influence sur l'état de la connexion, le taux de transfert courant et le délai global.
+Plus la taille de la mémoire tampon est faible, plus la probabilité d'un voyant rouge dans l'indicateur d'état (abandons) est élevée et plus le taux de transfert est élevé et le retard global est faible.
+
+![Dépendances du Délai de temporisation](https://user-images.githubusercontent.com/20726856/97361878-816a6680-18a0-11eb-8c1f-fbf956be4b81.png)
+
+Le réglage de la mémoire tampon est donc un compromis entre la qualité audio et le délai global.
+
+### Tampon de gigue avec indicateur de statut
+
+![Tampon de gigue](https://user-images.githubusercontent.com/20726856/97361956-9b0bae00-18a0-11eb-85de-dc2ded891175.png)
+
+Le tampon de gigue compense les gigues de synchronisation du réseau et de la carte son. La taille de la mémoire tampon influence donc la qualité du flux audio (le nombre d'interruptions) et le retard global (plus le tampon est long, plus le délai est important).
+
+Vous pouvez définir manuellement la taille du tampon de gigue pour le client local et le serveur distant. Pour la mémoire tampon de gigue locale, les pertes de flux audio sont indiquées par le voyant lumibaux situé sous les curseurs de taille de la mémoire tampon de gigue.
+Si le voyant devient rouge, un dépassement ou une sous-utilisation de la mémoire tampon a eu lieu et le flux audio est interrompu.
+
+Le réglage du tampon de gigue est donc aussi un compromis entre la qualité audio et le délai global.
+
+Si le paramètre Auto est activé, les tampons de gigue du client local et du serveur distant sont automatiquement réglés en fonction des mesures de la gigue de synchronisation du réseau et de l'interface audio. Si la vérification automatique est activée, les curseurs de la taille des tampons de gigue sont désactivés (ils ne peuvent pas être déplacés avec la souris).
+
+### Canaux Audio
+
+![Canaux Audio](https://user-images.githubusercontent.com/20726856/97362020-b24a9b80-18a0-11eb-8a55-ee536b29460b.png)
+
+Sélectionne le nombre de canaux audio à utiliser pour la communication entre le client et le serveur. Trois modes sont disponibles :
+
+Les modes Mono et Stéréo utilisent respectivement un et deux canaux audio.
+
+Entrée Mono/Sortie Stéréo : Le signal audio envoyé au serveur est mono mais le signal de retour est stéréo. C'est utile si l'interface audio a l'instrument sur un canal d'entrée et le micro sur l'autre. Dans ce cas, les deux signaux d'entrée peuvent être mélangés sur un canal mono mais le mixage du serveur est entendu en stéréo.
```

Comment:
* "_Entrée mono/sortie stéréo _" (no need for capital letters here)
* "_Le signal audio_" -> "_le signal audio_"
* "_l'interface audio_" -> "_la carte son_"